com.unity.render-pipelines.core/Runtime/Debugging/DebugUI.Fields.cs

```diff
@@ -320,6 +320,9 @@ internal void InitQuickSeparators()
 
             internal void InitIndexes()
             {
+                if (enumNames == null)
+                    return;
```

Comment:
This will cause init miss of 'indexes'. Init 'enumNames' by default with an empty array has less chance to produce other nullref errors.